### ISSUE TITLE
ArrayIndexOutOfBoundsException at `ModifierWithoutDefaultDetector.visitComposable`

### ISSUE THREAD

**Issue by jsjeon** [slackhq/compose-lints#423]
From https://issuetracker.google.com/371140566

Version of Gradle Plugin: 8.2.2
Version of Gradle: 8.9
android.experimental.lint.version=8.8.0-alpha04

The code example:
```kt
/**
 * Automatically requests focus after initial composition.
 */
@Suppress("ModifierComposable") // Comment
@Composable
fun Modifier.focusAutoRequester(): Modifier {
    val focusRequester = remember { FocusRequester() }

    LaunchedEffect(Unit) {
        focusRequester.requestFocus()
    }

    return focusRequester(focusRequester)
}
```

stacktrace:
```
Stack: \\\`ArrayIndexOutOfBoundsException:Arrays$ArrayList.get(Arrays.java:4165)
←ModifierWithoutDefaultDetector.visitComposable(ModifierWithoutDefaultDetector.kt:59)
←ComposableFunctionDetector$createUastHandler$1.visitMethod(ComposableFunctionDetector.kt:33)
←UElementVisitor$DispatchUastVisitor.visitMethod(UElementVisitor.kt:535)
←UElementVisitor$DelegatingUastVisitor.visitMethod(UElementVisitor.kt:818)
←UMethod.accept(UMethod.kt:42)
←ImplementationUtilsKt.acceptList(implementationUtils.kt:15)
←AbstractKotlinUClass.accept(AbstractKotlinUClass.kt:213)
←ImplementationUtilsKt.acceptList(implementationUtils.kt:15)
←UFile.accept(UFile.kt:89)
←UastLintUtilsKt.acceptSourceFile(UastLintUtils.kt:953)
←UElementVisitor$visitFile$3.run(UElementVisitor.kt:216)
←LintClient.runReadAction(LintClient.kt:1781)
←LintDriver$LintClientWrapper.runReadAction(LintDriver.kt:2747)
←UElementVisitor.visitFile(UElementVisitor.kt:213)
←LintDriver$visitUastDetectors$1.run(LintDriver.kt:2048)
←LintClient.runReadAction(LintClient.kt:1781)
←LintDriver$LintClientWrapper.runReadAction(LintDriver.kt:2747)
←LintDriver.visitUastDetectors(LintDriver.kt:2048)
←LintDriver.visitUast(LintDriver.kt:2010)
←LintDriver.runFileDetectors(LintDriver.kt:1284)
←LintDriver.checkProject(LintDriver.kt:1070)
←LintDriver.checkProjectRoot(LintDriver.kt:622)
...
```

**Comment by ZacSweers** [comment]
Thanks!

**Comment by ZacSweers** [comment]
This was essentially fixed by #410 and available since 1.4.0
